resources/externs/react-tooltip.js

```diff
@@ -0,0 +1,116 @@
+/**********************************************************************
```

Comment:
This externs file is never called up in `prod.cljs.edn`. You should be able to remove it since you use `gobject/get` for the actual access below

Reply:
huh, so i don't. Tested in prod compile just to be sure

Reply:
e44863ed10753b90e3b7d28b7f16c2bb6c758841